test/test_multiprocessing.py

```diff
@@ -447,7 +492,6 @@ def test_event(self):
             self.assertEqual(list(tensor), [4, 4, 4, 4])
         p.join()
 
-    @staticmethod
```

Comment:
This looks like a botched merge. These should be static methods...

Reply:
yes